### ISSUE TITLE
Refine Public Interface or Interfaces

### ISSUE THREAD

**Issue by Danziger** [helloswyg/autoshapes-example#2]
The main interface (for React) might look something like:

    <Shape className="..." type="curyvy" orientation="vertical" padding={ 32 } />

But maybe some wrapper on top of that and/or different interfaces could be helpful, such as:

    <Shape className="..." type="curyvy" orientation="vertical" padding={ 32 } repeat={ [8. 16] } />
    <Shapes className="..." config={ [{ ... }, { ... }] } />

On the other hand, we need to decide what properties and values are we going to support and if a higher level abstraction could be helpful for some or some combinations of them.

Some additional considerations:

- If the size (`width` and `height`) of the SVG is set to `1em`, the user will be able to change the size using the `font-size` CSS property, without having to pass that as a prop.
- If the color of the SVG is set to `currentColor`, the user will be able to change it using the `color` CSS property, without having to pass that as a prop.
- If the shape area has paddings, we might want to respect that.
- Being able to set individual classes for each shape could be useful if the user wants more control to do something like animations or fancy rich data visualization (like a shape-based heatmap).
- The user might want to set a global theme for the shapes. Still, this can always be done spreading a constant that contains the common properties, so would be low prio. In any case, it can be done with a Provider.
- There's always a random component when generating the shapes. The user could "freeze" that using a `seed` property.

**Comment by Danziger** [comment]
@vincentropy I'll leave it to you to document the current (low-level I guess) interface in the README and link it here, for now I just added some things we could consider when tweaking them.

**Comment by vincentropy** [comment]
@Danziger I updated the public interface and expanded the readme to describe how it (currently) works. 

- I disabled themes for now, until we can figure out how to do this properly (see #5)
- the current interface enables drawing specific shapes inside a react component
  - proposed "repeat" will be available through the "complexity" parameter
  - proposed "orientation" is available through the transform style parameter (i.e. by setting rotation)
  - in principle size can also be set through the style parameters but I have not yet changed the default 
- some parameters still need to be implemented. see #7 #6 

See #4 for all changes.

I'm splitting some of the suggestions from above into separate issues. I would propose having separate issues for styling #8 and themes #5.
